network/bluetooth_att.ksy

```diff
@@ -0,0 +1,117 @@
+meta:
+  id: bluetooth_att
+  title: Bluetooth ATT (IEEE 802.15.1)
+  xref:
+    ieee: 802.15.1
+  license: CC0-1.0
+  ks-version: 0.8
+  endian: le
+doc: |
+  Bluetooth ATT is a part of the Bluetooth specification.
+  Attribute Profile, page 2288
+doc-ref: https://www.bluetooth.org/docman/handlers/downloaddoc.ashx?doc_id=457080
+
+seq:
+  - id: opcode
+    type: u1
+    enum: att_opcode
+    doc: page 2330
```

Comment:
doc-ref

Reply:
I don't catch it. The doc is a huge PDF file. It is referenced above. How'd I do it better?

Reply:
`doc-ref` allows to specify a page.